ui/pages/confirmations/components/confirm/smart-account-tab/smart-account-tab.test.tsx

```diff
@@ -42,6 +42,6 @@ describe('SmartAccountTab', () => {
     const { getAllByText } = renderComponent();
 
     expect(getAllByText('Smart Account')).toHaveLength(2);
-    expect(getAllByText('Standard Account')).toHaveLength(2);
+    expect(getAllByText('Standard account')).toHaveLength(2);
```

Comment:
By the assertion of this test it may seem like "Smart Account" and "Standard Account" appear next to each other in tabs if "Smart Account" is a special term should "Standard Account" be as well or is it fine to see "Smart Account" | "Standard account" 🤔 cc @coreyjanssen